### ISSUE TITLE
Deleting a variable in the Variable explorer.

### ISSUE THREAD

**Issue by DavidMArr** [spyder-ide/spyder#11999]
### What steps will reproduce the problem?

I just deleted a variable in the Variable explorer. Nothing more.

### Traceback
```python-traceback
  File "/home/david/anaconda3/lib/python3.7/site-packages/spyder/plugins/variableexplorer/widgets/collectionseditor.py", line 791, in contextMenuEvent
    self.refresh_menu()
  File "/home/david/anaconda3/lib/python3.7/site-packages/spyder/plugins/variableexplorer/widgets/collectionseditor.py", line 707, in refresh_menu
    self.refresh_plot_entries(index)
  File "/home/david/anaconda3/lib/python3.7/site-packages/spyder/plugins/variableexplorer/widgets/collectionseditor.py", line 712, in refresh_plot_entries
    is_list = self.is_list(key)
  File "/home/david/anaconda3/lib/python3.7/site-packages/spyder/plugins/variableexplorer/widgets/collectionseditor.py", line 1459, in is_list
    return self.var_properties[name]['is_list']
KeyError: 'vector'
```

## Versions

* Spyder version: 4.1.1 
* Python version: 3.7.4
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Linux 5.3.0-42-generic

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.3.0 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
keyring                        :  None (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.4.4 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.1 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.3 (OK)
sphinx >=0.6.6                 :  2.4.0 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
xdg >=0.26                     :  0.26 (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.15 (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  1.0.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)
```

**Comment by ccordoba12** [comment]
Duplicate of #11963

**Comment by ccordoba12** [comment]
It'll be fixed in our next version (4.1.2), to be released next week.
